loopback-2.x/client/public/js/applications.js

```diff
@@ -1,3 +1,6 @@
+// Copyright Owner 2015. All Rights Reserved.
```

Comment:
Please update `package.json` to contain `IBM Corp.` as author in order to have it generated properly. Thanks.